apps/extension/src/offscreen.ts

```diff
@@ -49,16 +47,16 @@ export const buildActionHandler = (request: ActionBuildRequest) => {
 };
 
 const spawnWorker = (
-  transactionPlan: JsonObject & { actions: Jsonified<Action>[] },
+  transactionPlan: JsonObject & { actions: JsonValue[] },
   witness: JsonObject,
   fullViewingKey: string,
   actionPlanIndex: number,
-): Promise<Jsonified<Action>> => {
+): Promise<JsonValue> => {
   return new Promise((resolve, reject) => {
     const worker = new Worker(new URL('./wasm-build-action.ts', import.meta.url));
 
     const onWorkerMessage = (e: MessageEvent) => {
-      resolve(e.data as Jsonified<Action>);
+      resolve(e.data as JsonValue);
```

Comment:
Same Q here

Reply:
events arriving here are technically 'unknown' and must be inspected with a type guard